electrum/gui/qt/three_keys_windows.py

```diff
@@ -96,6 +100,48 @@ def do_pay(self):
         self.wallet.set_alert()
         self.do_pay_invoice(invoice)
 
+    def pay_onchain_dialog(self, inputs, outputs, invoice=None, external_keypairs=None):
+        # trustedcoin requires this
+        if run_hook('abort_send', self):
+            return
+        is_sweep = bool(external_keypairs)
```

Comment:
All other occurences of is_sweep variable are set to false, as it causes problems with password-protected wallets. Eventually it will probably be completely removed.

Reply:
all occurrences of `is_sweep =` set to False